spec-0007/index.md

```diff
@@ -161,62 +177,106 @@ def check_random_state(seed=_NoValue, rng=_NoValue):
         return seed
 
     raise ValueError(f"'{seed}' cannot be used to seed a numpy.random.RandomState"
-                     " instance")
-
-
-def _prepare_rng(old_name, dep_version=None):
+                     " instance.")
+
+
+def _prepare_rng(old_name, position_num=None, dep_version=None):
+    """ Example decorator to deprecate old PRNG usage
+
```

Comment:
Maybe we need a paragraph here to describe what it does, given that very few people are likely to read the code.

Reply:
Added. Please check me.
You asked yesterday whether I thought it was a good thing to include the code. I said yes then because it helps projects, but another benefit is that it forces us to think through the implications of the SPEC very carefully.